corehq/apps/export/models/new.py

```diff
@@ -1075,10 +1080,11 @@ def _get_app_build_ids_to_process(domain, last_app_versions):
         return map(lambda app_build_version: app_build_version.build_id, app_build_verions)
 
     @staticmethod
-    def generate_schema_from_builds(domain, case_type, force_rebuild=False):
+    def generate_schema_from_builds(domain, app_id, case_type, force_rebuild=False):
```

Comment:
yeah so i was a bit torn, this is added here for a couple reasons:

1. this is the specific app that the user chose to have a case export for, so more likely than not they'd like to see an up to date version of the export for the case for that app.
2. it allows us to DRY up this code a bit (done in later commit)

the other apps are fetched from this function: https://github.com/dimagi/commcare-hq/blob/master/corehq/apps/export/models/new.py#L1071 which gets all app build ids in a domain. the reason i didn't include all live apps is that would mean that each time you open an export you have to process every single live app in a domain which changes more frequently than the live application. could be convinced otherwise though. i think for now it might be best to only support it if people complain? don't think it'll be hard to add in later